src/Email/Attributes/Response/Property/EmailListCollection.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Email\Attributes\Response\Property;
+
+use OpenApi\Attributes\Items;
+use OpenApi\Attributes\Property;
+use Pimcore\Bundle\StudioBackendBundle\Email\Schema\EmailLogEntry;
+
+/**
+ * @internal
+ */
+final class EmailListCollection extends Property
```

Comment:
We could use [Pimcore\Bundle\StudioBackendBundle\OpenApi\Attributes\Property](https://github.com/pimcore/studio-backend-bundle/blob/1.x/src/OpenApi/Attributes/Property/GenericCollection.php) instate?

Reply:
Did not see it, thanks! :) we can definitely use it